apis/cluster/serverpool.go

```diff
@@ -35,6 +35,7 @@ type ServerPool struct {
 	Image             string
 	Size              string
 	BootstrapScript   string
+	VPNScript         string
```

Comment:
So this PR is *fantastic* but I want to make sure we treat our API very delicately. Even though this very well might get refactored completely moving forward, we still should consider our changes carefully.

I love the idea of having the VPN logic pulled out into different scripts, and then appending them into the `customdata` in the droplet resource (what you have done here) but I am wondering if having a new member called `VPNScript` is the right approach.

Would it be better if we changed `bootstrapScript` -> `bootstrapScripts` and made it of type `[]string` instead?

We could then guarantee they are ordered correctly (using a proper `for` loop and index) when we append them.

This would be an easy change, and would allow us to add many scripts at runtime, instead of building in new members for every new type of script. Thoughts?

If we changed the API we would have to refactored the AWS implementation as well. It shouldn't be hard, and at this point I am *okay* with breaking the API because we aren't advertising this to be used yet.